core/modules/syncer.js

```diff
@@ -64,23 +64,27 @@ function Syncer(options) {
 	this.filterFn = this.wiki.compileFilter(this.wiki.getTiddlerText(this.titleSyncFilter));
 	// Record information for known tiddlers
 	this.readTiddlerInfo();
+	this.titlesToBeLoaded = {}; // Hashmap of titles of tiddlers that need loading from the server
+	this.titlesHaveBeenLazyLoaded = {}; // Hashmap of titles of tiddlers that have already been lazily loaded from the server
+	// Timers
 	this.taskTimerId = null; // Timer for task dispatch
 	this.pollTimerId = null; // Timer for polling server
+	// Number of outstanding requests
 	this.numTasksInProgress = 0;
```

Comment:
Why don't you rename the variable to `numOpenRequests` or something similar?

Reply:
Good thinking

Reply:
Actually, the code uses "task" quite a few times to refer to the CRUD transactions, so I think it's consistent enough.

Reply:
ok